api_tests/btsieve/bitcoin_only/test.ts

```diff
@@ -25,9 +25,9 @@ setTimeout(async function() {
         describe("Bitcoin", () => {
             describe("Transactions", () => {
                 it("btsieve should respond not found when getting a non-existent bitcoin transaction query", async function() {
-                    const res = await request(btsieve.url()).get(
-                        "/queries/bitcoin/regtest/transactions/1"
-                    );
+                    const res = await request(btsieve.url())
```

Comment:
I'd try to override this request with one that do a set of the header.
```
const request(url: string) {
return Request.request(url).set("Expected-Version", btsieve.expectedVersion);
}
Note: I assume the `.set()` can get done before `.get()` call, if not, it's a bit harder.

Reply:
I did try, but `.set()` can only be done after `.get()` :disappointed: